### PR TITLE
fix: add mender-setup to README

### PR DESCRIPTION
**Pull request by TheYoctoJester**
With the mender-setup tool being factored out, somebody who just follows the instructions in the README has no way to obtain a working configuration.
Hence, add a reference to the repository and mention that other configuration flows exist.

Changelog: Title
Ticket: None

# External Contributor Checklist

🚨 Please review the [guidelines for contributing](https://github.com/mendersoftware/mender/blob/master/CONTRIBUTING.md) to this repository.

- [ ] Make sure that all commits follow the conventional commit [specification](https://www.github.com/mendersoftware/mendertesting/commitlint/grammar.md) for the Mender project.

The majority of our contributions are fixes, which means your commit should have
the form below:

```
fix: <SHORT DESCRIPTION OF FIX>

<OPTIONAL LONGER DESCRIPTION>

Changelog: <USER-FRIENDLY-CHANGE-DESCRIPTION> or <None>
Ticket: <TICKET NUMBER> or <None>
```

- [ ] Make sure that all commits are signed with [`git --signoff`](https://git-scm.com/book/en/v2/Git-Tools-Signing-Your-Work). Also note that the signoff author must match the author of the commit.

### Description

Please describe your pull request.

Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -215,9 +222,6 @@ steps above:
   "ServerCertificate": "/etc/mender/server.crt"
 ```
 
-Keep in mind that `/etc/mender/mender.conf` will be overwritten if you rerun the
```
Comment (oleorhagen) [inline README.md:218]: Q: Why remove this? Is it not true anymore?

**Review by oleorhagen — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (TheYoctoJester) [reply README.md:218]: Yep, it does not get installed anymore, at least in none of my tests.

Comment (oleorhagen) [reply README.md:218]: Yes, but split it out into a separate commit please :)

**Comment by TheYoctoJester** [comment]
@oleorhagen okay now?

**Comment by oleorhagen** [comment]
Yes, looks good now! :)

Just one last thing. Your commits should not be fixes. This should not be a patch for the client ;)

Make them `chores`

**Comment by TheYoctoJester** [comment]
> Yes, looks good now! :)
> 
> Just one last thing. Your commits should not be fixes. This should not be a patch for the client ;)
> 
> Make them `chores`

Done.

**Comment by oleorhagen** [comment]
Looks good!

@lluiscampos will you force-merge this? :pray:

**Comment by oleorhagen** [comment]
Oh, actually, one last thing. No `Changelog` needed for this @TheYoctoJester . Sry, I missed it the first time around

**Pull request merged**